docs/fieldnorm/software_life_cycle_processes.md

```diff
@@ -0,0 +1,104 @@
+3. A well-defined development and release cycle with set time periods for preparation, development, testing, and releasing. 
+   This allows multiple developers to work cohesively on the project and creates structured testing and release moments during the implementation and maintenance processes. It also enables ample opportunities for user reflection on the deliverables during the lifetime of projects.
+4. Writing code with unit and integration tests. Moreover, conducting regular code reviews is highly recommended. These methods will be invaluable during software maintenance, testing, and release. It enables a wider understanding of parts of the code, as unit tests often embody general usage as well as very specific exceptions, and code reviews enforce colleagues to have a look and understand what your code is doing.
+
+## 5. Testing
+Before implementation, software should be extensively tested, verified, and validated. 
+While ISO 15189 does not provide specific guidelines on how to conduct software validation, 
+it emphasizes the importance of ensuring that software is validated and verified following recognized industry standards. 
+Software validation and verification should be performed in accordance to this document [Software Verification & Software Validation](verification_validation.md).
+
+## 6. Implementation
+Once everything is ready, there are a few key processes that should be included in guidelines to facilitate the release of the software:
+1. The process for establishing a release. The exact time point at which software is released could be determined 
+   by a Development & Release cycle or be released in an independent manner. 
+2. The process for performing a release should be well-defined. 
+3. The software functionality and release date should be clearly communicated to end users. 
+
+## 7. Maintenance
+Maintenance primarily centers around fixing bugs, monitoring the software, introducing new features, 
+and upgrading to newer versions with new requirements. 
+The process follows the same procedures as the development, testing, and implementation phases. 
+Only certain maintenance processes that deviate from the normal procedures for development, testing, 
+and implementation should be documented (hotfixes, for example).
+
+## Hotfixes
+Hotfixes are updates which are directly uploaded in to the actual live software without going through the development, 
+testing and implementation phases of the SDLC. They are meant as a quick way to fix unforeseen but process breaking bugs. 
+In general, having well-defined development, testing and implementation processes would negate the need for hotfixes. However, there is always the possibility for bugs to occur. The laboratory should have clear guidelines on how and when hotfixes are performed. 
+
+> 📝 _Note: hotfixes could be classified as non-conforming work and might need to be followed up through the laboratories QMS, more on this in [user feedback](feedback.md)_
```

Comment:
See comment above

Reply:
Yup we can remove both. However on the webpages the emoji looks nice, and in general looking around by similar pages emojis are added to a boxes to indicate whether it is additional information, a warning or for example an assignment. But if we do it here we should do this also on the other pages.